cloud/google/machineactuator.go

```diff
@@ -376,12 +380,12 @@ func (gce *GCEClient) PostCreate(cluster *clusterv1.Cluster, machines []*cluster
 		return fmt.Errorf("error creating default storage class: %v", err)
 	}
 
-	err = gce.CreateIngressControllerServiceAccount(cluster, machines)
```

Comment:
The `GCEClient` type potentially implements the `machineDeployer` interface [1]. Removing the `machines []*clusterv1.Machine` parameter would break it. Not sure if there is any document/write-up what would reason about the need to pass list of machines into abstract "PostCreate" operation.

[1] https://github.com/kubernetes-sigs/cluster-api/blob/5997a0180f13725ad1f6769e8263c426b15d2d96/gcp-deployer/deploy/machinedeployer.go#L25

Reply:
removed, and removed from the interface as well